apps/www/_blog/2023-08-09-supabase-studio-3-0.mdx

```diff
@@ -0,0 +1,149 @@
+
+A few months back, we saw this [PR](https://github.com/supabase/supabase/pull/13745) come in out of the blue from [HTMHell](https://github.com/HTMHell). They built the entire thing with zero help or direction from our team. We were blown away. We had some changes to make on the backend to properly accommodate the UI, and now we're almost ready to get this out into the wild!
+
+![Role management](/images/blog/launch-week-8/day-3/role-management.png)
+
+Due to the security focus of this feature, we want to make sure we do a very thorough job of testing, so we're hoping to make this generally available in the next week or so.
+
+Massive thanks to [HTMHell](https://github.com/HTMHell) (amazing handle btw) for the work on this!
+
+## Shared SQL Snippets
+
+Speaking of commonly requested features, this one has to be in the all-time top 5. Your beautiful, hand-crafted SQL snippets used to be yours and yours alone. Now you can share them with team members and let them bask in your technical prowess.
+
+![Share SQL Snippets](/images/blog/launch-week-8/day-3/share-query.png)
+
+You can create a set of project-wide snippets for doing common tasks, making it faster to collaborate and build. To share a snippet, just take a personal snippet that ~~Supabase AI~~ you wrote and share it with the project. It will show up in a new Project Snippets list that's visible to everyone on the team.
+
+Teamwork makes the dream work!
+
+## Database Migration UI
+
+We're releasing a new UI for working with database migrations right from the Studio. Database migrations give you a way to update your database using version-controlled SQL files. They describe changes that you want to make to your database, and also keep track of what changes have been made to your database over time.
+
+As migrations get run against your project from the CLI, you can see information in the Studio about when the migration was run, by who and what changes were made. [See the documentation](https://supabase.com/docs/guides/cli/local-development#database-migrations) to get started with migrations.
+
+![Migrations UI](/images/blog/launch-week-8/day-3/migrations.png)
+
+## Wrappers UI
+
+During Launch Week 6, we [announced](https://supabase.com/blog/postgres-foreign-data-wrappers-rust) Supabase Wrappers — a framework for building foreign data wrappers with Postgres. Wrappers allow your Supabase project to act as a one-stop hub for your data.
+
+When we released Wrappers, we had support for just two providers — Stripe and Firebase. We're now up to 6! This round, we're happy to release support for S3, ClickHouse, BigQuery, and Logflare! Wrappers add a mind-bending level of extensibility to Supabase projects. You can pull data straight into your projects as though they were normal Supabase tables — you can even query them with our client libraries. It's a whole new world of possibilities.
+
+![Wrappers](/images/blog/launch-week-8/day-3/wrappers.png)
```

Comment:
Is this the correct image? Shouldn't we use one showcasing all the wrappers, like: 

<img width="1045" alt="image" src="https://github.com/supabase/supabase/assets/20600542/ef51b456-c7b1-4f6d-b7a8-221c1ecc2ad3">